src/twisted/test/test_defer.py

```diff
@@ -645,6 +645,121 @@ def test_cancelGatherResultsWithAllDeferredsCallback(self):
         self.assertEqual(callbackResult[1], "Callback Result")
 
 
+    def test_unbox(self):
+        """
+        L{defer.unbox} should unbox a Deferred that has already had it's
+        callback function called.
+        """
+        result = object()
+
+        d = defer.Deferred()
+        d.callback(result)
+
+        self.assertIs(defer.unbox(d), result)
+
+
+    def test_unboxRaisesFailureException(self):
+        """
+        L{defer.unbox} should raise the original exception if the result of the
```

Comment:
s/should raise/raises (and same with below)

I think generally AIUI we try to follow https://jml.io/pages/test-docstrings.html